### ISSUE TITLE
Trouble getting flow-storm running again

### ISSUE THREAD

**Issue by ingesolvoll** [jpmonettas/flow-storm#15]
I haven't had flow-storm running in a while, trying to get it to work again now is very difficult and I don't know what I'm doing wrong. I'm primarily using it for CLJS.

I'm running java 11 (openjdk), flow-storm-debugger 0.6.0

Using flow-storm 0.4.x in my app renders this message in my browser console and nothing ends up in debugger when I try to trace.
```
#error {:message "Invalid event", :data {:given "~:chsk/handshake", :errors {:wrong-type 
{…}
[:expected :vector]
[:actual {:type #object[String], :value "~:chsk/handshake"}]
```

Trying flow-storm-debugger 0.5.0 crashes on startup with this message:
```
clj -Sdeps '{:deps {jpmonettas/flow-storm-debugger {:mvn/version "0.5.0"}}}' -m flow-storm-debugger.main
WARNING: When invoking clojure.main, use -M
Syntax error (ClassNotFoundException) compiling at (cljfx/coerce.clj:1:1).
javafx.event.EventHandler
```

If I try to run flow-storm 0.5.0, I'm getting this on compiling clojurescript:

```
WARNING: Use of undeclared Var cljs.core.async/go-loop at line 150 /Users/inge/.boot/cache/tmp/Users/inge/dev/aria/m3c/-p1s4zp/js/main.out/flow_storm/tracer.cljc
WARNING: Use of undeclared Var flow-storm.tracer/batch at line 150 /Users/inge/.boot/cache/tmp/Users/inge/dev/aria/m3c/-p1s4zp/js/main.out/flow_storm/tracer.cljc
WARNING: Use of undeclared Var flow-storm.tracer/last-send at line 151 /Users/inge/.boot/cache/tmp/Users/inge/dev/aria/m3c/-p1s4zp/js/main.out/flow_storm/tracer.cljc
WARNING: Use of undeclared Var flow-storm.tracer/last-send at line 157 /Users/inge/.boot/cache/tmp/Users/inge/dev/aria/m3c/-p1s4zp/js/main.out/flow_storm/tracer.cljc
WARNING: Use of undeclared Var flow-storm.tracer/batch at line 159 /Users/inge/.boot/cache/tmp/Users/inge/dev/aria/m3c/-p1s4zp/js/main.out/flow_storm/tracer.cljc
WARNING: Use of undeclared Var flow-storm.tracer/batch at line 165 /Users/inge/.boot/cache/tmp/Users/inge/dev/aria/m3c/-p1s4zp/js/main.out/flow_storm/tracer.cljc
Can't recur here at line 166 /Users/inge/.boot/cache/tmp/Users/inge/dev/aria/m3c/-p1s4zp/js/main.out/flow_storm/tracer.cljc
```

**Comment by jpmonettas** [comment]
Hi @ingesolvoll,
 
I think is better if you run flow-storm-debugger 0.6.0,  and add to your project flow-storm 0.5.0 which are the latest versions of both. Any other combination behavior is undefined.

It would be nice to have some good error message when versions are incompatible but I didn't have time to add that yet.

Please let me know if it works for you, if it doesn't I can help you debug it further.
I just created a empty cljs project from scratch with shadow-cljs and flow-storm 0.5.0 and everything compiles fine.

**Comment by ingesolvoll** [comment]
@jpmonettas Right, that's what I thought. Your stuff has been quite stable so far, so I figured the error had to be on my end. To begin with I'm actually using debugger 0.6.0 and lib 0.5.0, that's when I'm getting the `Use of undeclared Var` stuff when compiling. Probably some dependency conflict then, I'll go through them first.

**Comment by ingesolvoll** [comment]
Right, I figured it out. flow-storm gets its core.async dep as a transitive dep from sente. In my project core.async was explicitly excluded from our sente dep that was overriding flow-storm's sente dep. Once I removed that exclusion, everything works as it should.

flow-storm clearly requires core.async to be present, maybe it would be a good idea to explicitly depend on it? This won't happen to a lot of people, but it might happen again.
